### PR TITLE
Adding physicalInputReadTime

### PR DESCRIPTION
**Pull request by ankitdixit**

### CODE REVIEW

**Comment by ankitdixit** [comment]
Fixes https://github.com/prestosql/presto/issues/2232

**Comment by ankitdixit** [comment]
image

File: presto-main/src/main/java/io/prestosql/execution/QueryStats.java [hunk 1]
```diff
@@ -219,6 +221,7 @@ public QueryStats(
         this.physicalInputDataSize = requireNonNull(physicalInputDataSize, "physicalInputDataSize is null");
         checkArgument(physicalInputPositions >= 0, "physicalInputPositions is negative");
         this.physicalInputPositions = physicalInputPositions;
+        this.physicalInputReadTime = physicalInputReadTime;
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/execution/QueryStats.java]: require non null

File: presto-main/src/main/java/io/prestosql/execution/BasicStageStats.java [hunk 2]
```diff
@@ -114,6 +117,7 @@ public BasicStageStats(
         this.completedDrivers = completedDrivers;
         this.physicalInputDataSize = requireNonNull(physicalInputDataSize, "physicalInputDataSize is null");
         this.physicalInputPositions = physicalInputPositions;
+        this.physicalInputReadTime = physicalInputReadTime;
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/execution/BasicStageStats.java]: require non null

File: presto-main/src/main/java/io/prestosql/execution/StageStats.java [hunk 3]
```diff
@@ -176,6 +178,7 @@ public StageStats(
         this.physicalInputDataSize = requireNonNull(physicalInputDataSize, "physicalInputDataSize is null");
         checkArgument(physicalInputPositions >= 0, "physicalInputPositions is negative");
         this.physicalInputPositions = physicalInputPositions;
+        this.physicalInputReadTime = physicalInputReadTime;
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/execution/StageStats.java]: require non null

**Comment by sopel39** [comment]
This doesn't fix #2232 as it doesn't expose read rate (e.g MB/s), so I will leave that open

**Comment by sopel39** [comment]
merged, thanks!

**Pull request merged**